iina/MPVController.swift

```diff
@@ -342,8 +342,13 @@ class MPVController: NSObject {
     strArgs.insert(command.rawValue, at: 0)
     strArgs.append(nil)
     var cargs = strArgs.map { $0.flatMap { UnsafePointer<Int8>(strdup($0)) } }
+    defer {
```

Comment:
I'd like a consistent coding style with this kind of allocation. So I changed this on and the one in md5.